docs/content/usage/live-code.mdx

```diff
@@ -77,3 +77,41 @@ export default function LivePreviewWrapper({children}) {
   )
 }
 ```
+
+## No-Inline
```

Comment:
I don't think the "inline" needs to be capitalized since we use sentence case for our headings.

```suggestion
## No-inline
```

Reply:
Thanks... I don't think I'll ever get used to this 🙈